wordle-web/components/UserInput.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+	<v-dialog v-model="dialogBox" max-width="500" persistent>
+		<v-card>
+			<v-sheet color="primary">
+				<v-card-text>UserName: </v-card-text>
+			</v-sheet>
+			<v-form class="mx-3 mt-5">
+				<v-text-field v-model="usersNameInput" label="user" variant="outlined" clearable required>
+			    </v-text-field>
+			</v-form>
+			<v-card-actions class="mx-4 mb-3">
+				<v-spacer></v-spacer>
+				<v-btn color="success" @click="userSaving">SAVE</v-btn>
+			</v-card-actions>
+		</v-card>
+	</v-dialog>
+</template>
+
+<script setup lang="ts">
+import { ref } from "vue";
+import nuxtStorage from "nuxt-storage";
+const dialogBox = ref<boolean>(true);
+const usersNameInput = ref<string>('');
```

Comment:
When you initialize `usersNameInput` try setting it to 'Guest' it should display 'Guest' when no name is set.

Reply:
Done, thanks!